packages/core/src/pipeline/config/triggers/artifacts/ArtifactService.ts

```diff
@@ -11,4 +11,8 @@ export class ArtifactService {
       .query({ type: type, artifactName: artifactName })
       .get();
   }
+
+  public static getArtifactByContentReference(contentRef: string): PromiseLike<{ reference: string }> {
+    return REST(`/artifacts/content-address/${contentRef}`).get();
```

Comment:
I think the convention is to use `.path()` rather than String templates (see method above).

Reply:
So `contentRef` is `application/hash`, so the reason it was done this way is to avoid HTTP escaping the `/`.